automarker_2/automarker_app/lib/steps.py

```diff
@@ -764,17 +764,21 @@ def run(self, project_uri, clone_dir_path, self_test, config, fail_fast):
                 )
             else:
                 l = stderr.strip().split("\n")
-                if len(l) == 2:
-                    # It would look something like: ['[NbConvertApp] Converting notebook /home/sheena/workspace/Tilde/automarker_2/gitignore/247-python-perfect/notebooks/personality.ipynb to notebook', '[NbConvertApp] Writing 96572 bytes to /home/sheena/workspace/Tilde/automarker_2/gitignore/247-python-perfect/notebooks/personality.ipynb', ]
 
-                    if ("Converting notebook" not in l[0]) and ("Writing" not in l[1]):
-                        breakpoint()
-                        not_sure
+                # It would look something like: ['[NbConvertApp] Converting notebook /home/sheena/workspace/Tilde/automarker_2/gitignore/247-python-perfect/notebooks/personality.ipynb to notebook', '[NbConvertApp] Writing 96572 bytes to /home/sheena/workspace/Tilde/automarker_2/gitignore/247-python-perfect/notebooks/personality.ipynb', ]
 
-                else:
+                start_line_prefix = "[NbConvertApp] Converting notebook"
+                end_line_prefix = "[NbConvertApp] Writing"
+
+                start_line_present = any(
+                    line.startswith(start_line_prefix) for line in l
+                )
+                end_line_present = any(line.startswith(end_line_prefix) for line in l)
+
+                # We check for these lines only because there may be some warnings in stderr.
+                if not start_line_present or not end_line_present:
```

Comment:
Isn't this equivalent and easier to read?

```
if not(start_line_present and end_line_present):
```

We want the start and end to be present. If they are not both present then it means something funny is happening, right?